pkgs/applications/virtualization/xen/generic/default.nix

```
@@ -717,8 +721,9 @@ stdenv.mkDerivation (finalAttrs: {
 
         mainProgram = "xl";
 
-        # Evaluates to x86_64-linux.
-        platforms = lib.lists.intersectLists lib.platforms.linux lib.platforms.x86_64;
+        # Evaluates to [x86_64-linux aarch64-linux].
```

```suggestion
        # Evaluates to [ "x86_64-linux" "aarch64-linux" ].
```
`nixfmt-rfc-style` won't pick up the comment.